tests/unit_tests/test_wave_06.py

```diff
@@ -76,7 +76,13 @@ def test_swap_best_by_category():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
+    assert result
+    assert len(jesse.inventory) == 3
+    assert len(tai.inventory) == 3
+    assert [item_a, item_b, item_f] == tai.inventory
```

Comment:
Nice! I think this may be just my opinion, but since we want to check that Tai's inventory contains items a, b, f I think this assertion would read more easily as:
`assert tai.inventory == [item_a, item_b, item_f]`

Just my opinion, very minor nit pick. Overall, nice robust test assertions!